presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriterConfig.java

```diff
@@ -38,9 +50,11 @@
                 .withRowGroupMaxRowCount(rowGroupMaxRowCount)
                 .withDictionaryMaxMemory(dictionaryMaxMemory)
                 .withMaxStringStatisticsLimit(stringStatisticsLimit)
-                .withMaxCompressionBufferSize(maxCompressionBufferSize);
+                .withMaxCompressionBufferSize(maxCompressionBufferSize)
+                .withStreamLayout(getStreamLayout(streamLayoutType));
     }
 
+    @NotNull
```

Comment:
The @NotNull annotation is a follow up from my previous PR.

https://github.com/prestodb/presto/pull/16089#discussion_r640650619